ozaria/site/components/interactive/PageInteractive/draggableOrdering/index.vue

```diff
@@ -68,11 +70,9 @@
       </div>
 
       <ordering-slots
```

Comment:
Do we even need ordering slots anymore?  These were originally used to bundle draggable-slots together but with the new UX specs I think each of these can be ULs, one of which is draggable.  This will significantly simplify the implementation.

Reply:
We do not need ordering slots anymore. I think most of the internals of ordering slots have been removed. I am +100 for simplifying it!